ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/*"
```

Comment:
Sorry.
@priidikvaikla according to exclude the files which contain numbers.
Let's use `"**/!(*-[0-9])*"`
Thank you.